app/assets/src/components/ui/containers/data_tooltip.scss

```diff
@@ -20,7 +20,9 @@
   }
 
   &Subtitle {
-    font-weight: 700;
+    margin-bottom: 15px;
+    line-height: 1.5em;
+    font-size: 12px;
```

Comment:
Could we migrate to a typography mixin?

Reply:
Yes, I've converted all fonts in this file to mixins.